model-server/requirements.txt

```
@@ -42,9 +42,8 @@ nbconvert==5.5.0
 nbformat==4.4.0
 notebook==5.7.8
 # Don't override system apt installed numpy and opencv-python on Raspbian.
-numpy==1.17.3
-# opencv-python==4.1.2.30
-opencv-python
+numpy==1.17.3 ; platform_machine != 'armv7l'
```

```suggestion
numpy ; platform_machine != 'armv7l'
```
Wondering if for non-Pi/armv7l, if we should leave numpy version open to support other platform available versions.